### ISSUE TITLE
read-simulator error when using error model

### ISSUE THREAD

**Issue by nilchia** [ncsa/NEAT#127]
Hello,
When I try to use read-simulator with an error model created by `model-seq-err` it gave me  this error:
```
ValueError: cannot convert float NaN to integer
```

here is the confige file:
```
reference: 'H1N1.fa'
error_model: 'neat_seq.p.gz'
```
and the code I used to create the model:
```
 neat  model-seq-err -i H1N1.fastq.gz -q 33 -Q 42 -o neat_seq
 ```
I am using neat version 4.2.5

Is there something wrong with the input and config or something wrong with the tool?

Thanks

**Comment by joshfactorial** [comment]
Currently working on. I hope to fix this week.
________________________________
From: Amirhossein ***@***.***>
Sent: Saturday, October 5, 2024 3:01 AM
To: ncsa/NEAT ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ncsa/NEAT] read-simulator error when using error model (Issue #127)


Hello,
When I try to use read-simulator with an error model created by model-seq-err it gave me this error:

ValueError: cannot convert float NaN to integer


here is the confige file:

reference: 'H1N1.fa'
error_model: 'neat_seq.p.gz'


and the code I used to create the model:

 neat  model-seq-err -i H1N1.fastq.gz -q 33 -Q 42 -o neat_seq


I am using neat version 4.2.5

Is there something wrong with the input and config or something wrong with the tool?

Thanks

**Comment by joshfactorial** [comment]
This should be fixed in version 4.2.6, now.
